pkg/monitors/vsphere/monitor.go

```diff
@@ -0,0 +1,40 @@
+package vsphere
+
+import (
+	"context"
+	"time"
+
+	"github.com/signalfx/signalfx-agent/pkg/monitors"
+	"github.com/signalfx/signalfx-agent/pkg/monitors/types"
+	"github.com/signalfx/signalfx-agent/pkg/monitors/vsphere/model"
+	"github.com/signalfx/signalfx-agent/pkg/utils"
+	"github.com/sirupsen/logrus"
+)
+
+type Monitor struct {
+	Output types.Output
+	cancel func()
+}
+
+func init() {
+	monitors.Register(
+		&monitorMetadata,
+		func() interface{} { return &Monitor{} },
+		&model.Config{},
+	)
+}
+
+func (m *Monitor) Configure(conf *model.Config) error {
+	var ctx context.Context
+	ctx, m.cancel = context.WithCancel(context.Background())
+	log := logrus.WithField("monitorType", "vsphere")
+	r := newRunner(ctx, log, conf, m)
+	utils.RunOnInterval(ctx, r.run, time.Duration(conf.IntervalSeconds)*time.Second)
```

Comment:
Hardcode to 20s and add doc telling people that IntervalSeconds is a noop on this monitor.

Reply:
Done.